admin/templates/sales_analytics/products_usage.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'base_analytics.html' %}
+{% load static %}
+
+{% block title_analytics %}
+    <h1>OSF Products Usage</h1>
+{% endblock title_analytics %}
```

Comment:
FWIW, we explicitly ignore the newline rule in our flake8 config, as I don't think it significantly affects readability.